modules/javafx.graphics/src/main/java/javafx/scene/TraversalDirection.java

```diff
@@ -28,18 +28,19 @@
  * Specifies the direction of focus traversal.
  *
  * @since 24
+ * @see {@link Node#requestFocusTraversal(TraversalDirection)}
  */
 public enum TraversalDirection {
-    /** Moves focus downward. */
+    /** Indicates a focus change to the node below the currently focused node. */
     DOWN,
-    /** Moves focus left. */
+    /** Indicates a focus change to the node to the left of the currently focused node. */
     LEFT,
-    /** Moves focus to the next focusable Node, possibly traversing into the children of the current parent. */
+    /** Indicates a focus change to the next focusable Node, possibly traversing into the children of the current parent. */
     NEXT,
-    /** Moves focus to the previous focusable Node. */
+    /** Indicates a focus change to the previous focusable node. */
```

Comment:
`NEXT` talks about traversing children, does `PREVIOUS` need to mention it too? Could it traverse out of the children of the current parent, or is it not symmetric?

Reply:
I think the whole "children" thing can be dropped, since there is no NEXT_IN_LINE in the public API anymore.